From ad23bf7e4a57c57a361a138d5237fb1d0013d859 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 29 Oct 2014 16:49:24 +0800
Subject: [PATCH 0508/1074] MLK-9766-2: ARM: dts: imxsx-arm2: correct inclued
 file on *-ldo.dts

Correct the inclued file on imx6sx-17x17-arm2-ldo.dts and
imx6sx-19x19-arm2-ldo.dts.

Signed-off-by: Robin Gong <b38343@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2-ldo.dts |    2 +-
 arch/arm/boot/dts/imx6sx-19x19-arm2-ldo.dts |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2-ldo.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2-ldo.dts
index c32feff..f087029 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2-ldo.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2-ldo.dts
@@ -6,7 +6,7 @@
  * published by the Free Software Foundation.
  */
 
-#include "imx6sx-sdb.dts"
+#include "imx6sx-17x17-arm2.dts"
 
 &cpu0 {
 	arm-supply = <&reg_arm>;
diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2-ldo.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2-ldo.dts
index c32feff..c048e03 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2-ldo.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2-ldo.dts
@@ -6,7 +6,7 @@
  * published by the Free Software Foundation.
  */
 
-#include "imx6sx-sdb.dts"
+#include "imx6sx-19x19-arm2.dts"
 
 &cpu0 {
 	arm-supply = <&reg_arm>;
-- 
1.7.5.4

